src/test/java/edu/kit/hci/soli/test/controller/BookingsControllerTest.java

```diff
@@ -20,8 +24,15 @@
 @SpringBootTest
 @AutoConfigureTestDatabase
 public class BookingsControllerTest {
-    @Autowired private UserService userService;
-    @Autowired private BookingsController bookingsController;
+    @Autowired
+    private UserService userService;
+    @Autowired
+    private BookingsController bookingsController;
+    @Autowired
+    private BookingsService bookingsService;
+
+
+
```

Comment:
The previous formatting is more readable IMO.
If you disagree, please at least don't add so much unneeded whitespace